rero_ils/modules/documents/jsonschemas/documents/document_admin_metadata-v0.0.1.json

```diff
@@ -0,0 +1,168 @@
+{
+  "adminMetadata": {
+    "title": "Admin metadata",
+    "type": "object",
+    "propertiesOrder": [
+      "note",
+      "encodingLevel",
+      "source",
+      "descriptionModifier",
+      "descriptionLanguage",
+      "descriptionConventions"
+    ],
+    "properties": {
+      "note": {
+        "title": "Notes",
+        "description": "Destin\u00e9es aux professionnels. On y trouve plusieurs types d'information: notices \u00e0 d\u00e9truire, source du catalogage, notice sommaire d'acquisition (SOMACQ), etc.",
```

Comment:
Should this description be in English in the sources?

Reply:
Changed to:
`Administrative information intended for professionals.`